scipy/signal/ltisys.py

```diff
@@ -605,6 +610,11 @@ class TransferFunction(lti):
     state-space matrices) is very inefficient and may lead to numerical
     inaccuracies.
 
+    If a `TransferFunction` is passed in for `*system`, coefficients for
+    both the numerator and denominator should be specified in descending
+    exponent order (e.g. s^2 + 3s + 5 would be represented as [1, 3, 5]).
+    This specification also applies to the "array_like" argument option.
```

Comment:
This statement only applies to the `(numerator, denominator)` tuple, not the `TransferFunction` object.